doozer/doozerlib/cli/fbc.py

```diff
@@ -180,3 +186,170 @@ async def fbc_import(runtime: Runtime, from_index: Optional[str], dest_dir: str)
     """
     cli = FbcImportCli(runtime=runtime, index_image=from_index, dest_dir=dest_dir)
     await cli.run()
+
+
+class FbcRebaseCli:
+    def __init__(
+            self,
+            runtime: Runtime,
+            version: str,
+            release: str,
+            commit_message: str,
+            push: bool,
+            operator_nvrs: Tuple[str, ...]):
+        self.runtime = runtime
+        self.version = version
+        self.release = release
+        self.commit_message = commit_message
+        self.push = push
+        self.operator_nvrs = operator_nvrs
+        self._logger = LOGGER.getChild("FbcRebaseCli")
+        self._db_for_bundles = KonfluxDb()
+        self._db_for_bundles.bind(KonfluxBundleBuildRecord)
+
+    async def run(self):
+        runtime = self.runtime
+        logger = self._logger.getChild("run")
+        if runtime.images and self.operator_nvrs:
+            raise click.BadParameter("Do not specify operator NVRs when --images is specified")
+        runtime.initialize(config_only=True)
+        assembly = runtime.assembly
+        if assembly is None:
+            raise ValueError("Assemblies feature is disabled for this group. This is no longer supported.")
+        assert runtime.konflux_db is not None, "konflux_db is not initialized. Doozer bug?"
+        konflux_db = runtime.konflux_db
+        konflux_db.bind(KonfluxBuildRecord)
+
+        dgk_operator_builds = await self._get_operator_builds()
+        bundle_builds = await self._get_bundle_builds(list(dgk_operator_builds.values()), strict=False)
+        not_found = [dkg for dkg, bundle_build in zip(dgk_operator_builds.keys(), bundle_builds) if bundle_build is None]
+        if not_found:
+            raise IOError(f"Bundle builds not found for {not_found}. Please build the bundles first.")
+        bundle_builds = cast(List[KonfluxBundleBuildRecord], bundle_builds)
+
+        assert runtime.source_resolver is not None, "source_resolver is not initialized. Doozer bug?"
+        assert runtime.group_config is not None, "group_config is not initialized. Doozer bug?"
+
+        rebaser = KonfluxFbcRebaser(
+            base_dir=Path(runtime.working_dir, constants.WORKING_SUBDIR_KONFLUX_FBC_SOURCES),
+            group=runtime.group,
+            assembly=assembly,
+            version=self.version,
+            release=self.release,
+            commit_message=self.commit_message,
+            push=self.push,
+            upcycle=False,
+        )
+        tasks = []
+        for dkg, bundle_build in zip(dgk_operator_builds.keys(), bundle_builds):
```

Comment:
```suggestion
        for dgk, bundle_build in zip(dgk_operator_builds.keys(), bundle_builds):
```
and in the usage below it